snap/gui/musescore.desktop

```diff
@@ -0,0 +1,17 @@
+[Desktop Entry]
+Version=1.1
+Name=MuseScore
+GenericName=Music notation
+GenericName[de]=Notensatz
+GenericName[fr]=Notation musicale
+Comment=Create, play and print sheet music
+Comment[ru]=Визуальный редактор нотных партитур
+Comment[fr]=Gravure de partitions musicales
```

Comment:
We would say "_En_gravure de partitions musicales"

Reply:
Same 'mistake' as in build/Linux+BSD/mscore.desktop.in